010_sumAndMultiplyByN.js

```diff
@@ -0,0 +1,35 @@
+/* 10. Write a program that asks the user for a number n and gives them the possibility to choose between 
+computing the sum and computing the product of 1,…,n. */
+
+const operation = (choice) => {
+	if (choice == "m") {
+		MultiplyByN(number);
+	} else {
+		sumByN(number);
```

Comment:
Lot of issues with spacing. Contact Akhil

Reply:
I tried multiple times to correct the spacing. As suggested by Akhil added this line"?ts=2" to the URL (https://github.com/arjune64/vonnue-programming-arjun/pull/15?ts=2) and the issue is resolved. I think this is a compatibility issue.